app/pages/form/[id]/[page].tsx

```diff
@@ -29,12 +37,21 @@ const FormPage = ({ preloadedQuery }: any) => {
     }
   );
 
+  const trimApptitle = (title:string) => {
+    if (!Boolean(title)) return;
```

Comment:
@VB-at-Bis Nice job on this, it's working well. Just need to remove this `Boolean` causing `Error: Redundant Boolean call.  no-extra-boolean-cast` during project build.

Reply:
Fixed